GLOSSARY.md

```diff
@@ -32,3 +33,141 @@
+
+
+##### Digital Guide
+See _Exhibition guide_.
+
+
+##### Digital Location
+One of the _Location_ options, materials that are available online are identified as such. [Example of an online work](https://wellcomecollection.org/works/a222wwjt).
+
+
+##### Digitised Item
+
+
+##### Event
+[Example](https://wellcomecollection.org/events/Y7P3oxEAAHC43k0_)
+Displayed in various listing pages, including [Events](https://wellcomecollection.org/events)
+<!-- Add description -->
+
+
+##### Exhibition
+[Example](https://wellcomecollection.org/exhibitions/Yv95yBAAAILuCNv6)
+Displayed in various listing pages, including [Exhibitions](https://wellcomecollection.org/exhibitions)
+<!-- Add description -->
+
+
+##### Exhibition Guide
+[Example](https://www-stage.wellcomecollection.org/guides/exhibitions/Y2omihEAAKLNfLar)
+A Prismic _Content type_ that allows the creation of online exhibition guides in multiple formats, such as BSL videos or audio tours.
+Also referred to as _Digital Guide_.
+
+
+##### Format
+The options change based on the content type, but for Stories (Articles), for example, it could be Webcomic, Long Read, In Picture&hellip; For an Event, it could be Game, Screening, Workshop&hellip;. They are sometimes referred to as the "type" of [insert content type name].
+
+
+##### Image Gallery
+A component that can be added in a Story (Article). These articles will most likely have the _format_ "In Pictures".
+
+
+##### In Pictures
+A _format_ for a Story (Article) that most likely contains an Image Gallery component.
+
+
+##### Installation
+
+
+##### License/Licence
+Tells you how you're able to use the material. For example; [PDM](https://creativecommons.org/publicdomain/mark/1.0/), [CC-BY-NC-ND](https://creativecommons.org/licenses/by-nc-nd/2.0/).
+Spelled Licence on the UI (UK spelling) and License in the API.
+
+
+##### Location
+Where the item can be found. Could be on the open shelves (readily available to readers), or in _closed stores_, as well as online (known as _digital location_).
+
+
+##### Manuscript
+[Example](https://wellcomecollection.org/works/a29c8r26)
+A work _format_ (twinned with _archive), it is for hand-written material.
+
+##### Photo Story
+A Story (Article) content type.
+<!-- Add description -->
+
+
+##### Role
+Part of the _Contributors_ object, defines the type of _contributor_ (author, illustrator, etc.)
+
+
+##### Season
+A season is a grouping of most any _content types_, but primarily events, exhibitions, stories. The idea is that it's a time-limited, curated programme of activities, and it gets promoted as one programme. e.g.: [What does it mean to be human, now?](https://wellcomecollection.org/seasons/X84FvhIAACUAqiqp). They are displayed on `/seasons/[id]`.
+
+
+##### Serial
+A serial is a type of _series_, in which _articles_ are scheduled to be published.
+
+
+##### Series
```

Comment:
Events can be grouped into event series: a grouping of events under a regularly occurring singular title/theme (e.g. Exploring Research, Study Days)

Articles can be grouped into story series: a grouping of articles connected thematically. These may be guest edited and include articles written by multiple contributors (unlike a _serial_ which tends to have the same contributor(s) across all articles in the serial).